packages/unified-latex-to-pretext/libs/pre-conversion-subs/macro-subs.ts

```diff
@@ -1,25 +1,39 @@
-import { xcolorMacroToHex } from "@unified-latex/unified-latex-ctan/package/xcolor";
 import { htmlLike } from "@unified-latex/unified-latex-util-html-like";
 import * as Ast from "@unified-latex/unified-latex-types";
 import { getArgsContent } from "@unified-latex/unified-latex-util-arguments";
 import { printRaw } from "@unified-latex/unified-latex-util-print-raw";
 import { VisitInfo } from "@unified-latex/unified-latex-util-visit";
+import { VFile } from "unified-lint-rule/lib";
+import { s } from "@unified-latex/unified-latex-builder";
+import { VFileMessage } from "vfile-message";
 
 /**
  * Factory function that generates html-like macros that wrap their contents.
  */
 function factory(
     tag: string,
+    isWarn = false,
```

Comment:
-> `shouldWarn`

Reply:
Describe with docstring what this does

Reply:
Changing this to a warning message itself would make it a lot more explicit what is happening.